projects/ngx-formentry/src/components/number-input/number-input.component.html

```diff
@@ -29,8 +30,22 @@
       [attr.step]="step"
       [disabled]="disabled"
       [required]="required"
+      [attr.data-invalid]="invalid ? invalid : null"
+      [placeholder]="placeholder"
       (input)="onNumberInputChange($event)"
     />
+    <svg
```

Comment:
I've updated the `NumberInput` component API to match the related [Carbon component](https://angular.carbondesignsystem.com/?path=/docs/components-number-input--docs) where the code was originally obtained from.